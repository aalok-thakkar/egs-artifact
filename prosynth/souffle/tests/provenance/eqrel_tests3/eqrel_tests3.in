explain isEqrel(1, 1)
explain notEqrel(1, 1)
