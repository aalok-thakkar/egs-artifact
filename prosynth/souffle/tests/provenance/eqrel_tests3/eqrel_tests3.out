isEqrel(0, 1)                
----------(R3)               
isEqrel(1, 0)  isEqrel(0, 1) 
-------------------------(R2)
        isEqrel(1, 1)        
isEqrel(0, 1)                 
----------(R3)                
isEqrel(1, 0)  isEqrel(0, 1)  
-------------------------(R2) 
        isEqrel(1, 1)         
--------------------------(R1)
        notEqrel(1, 1)        
